src/sentry/integrations/slack/message_builder/notifications.py

```diff
@@ -61,6 +67,7 @@ def build(self) -> SlackBody:
 
         if isinstance(self.notification, ReleaseActivityNotification):
             return self._build(
+                text=" ",
```

Comment:
If `text` is actually required, we should also undo the logic that omits it from the payload.